### ISSUE TITLE
5.5:Preview window does not reload automatically with post updates

### ISSUE THREAD

**Issue by ddryo** [WordPress/gutenberg#24562]
Thank you for your development.
With the 5.5 update, Gutenberg is even better.

However, I have one thing to ask, so I would like to ask you a question.

The preview window opened in "Preview in new tab" no longer reloads automatically when the post is updated.

Is this normal behavior from 5.5?
Is it changed for some reason?

I'm sad that automatic reload was so convenient.

If it's a bug, I hope it will be fixed...

**Comment by Computerbuddha** [comment]
Same issue here. Hope it´s a bug. Also now you have two (instead of one) clicks for the preview: 1. "Preview" -> Popup -> 2. "Preview in a new tab".

**Comment by talldan** [comment]
@ddryo Thanks for reporting this, it does indeed seem to be a bug/regression in v5.5. I think the problem is the behavior is tied to the visibility of the 'Preview in new Tab' button. If you leave the new Preview menu open, then save a change, you'll notice the preview does update like it used to.

What seems to be happening is that `previewWindow` is `undefined` here:
https://github.com/WordPress/gutenberg/blob/master/packages/editor/src/components/post-preview-button/index.js#L131-L132

The component must be losing it's reference to `previewWindow` when umounted.

@Computerbuddha There's a separate issue for the two clicks - https://github.com/WordPress/gutenberg/issues/24584.

**Comment by Mamaduka** [comment]
Should we close this issue in favor of #24584?

It has been almost five years, and it appears that this regression is not a definitive feature.

**Comment by talldan** [comment]
Agreed, unfortunately there's probably not much chance of this feature coming back unless someone decides to work on it.
